.travis.yml

```diff
@@ -13,8 +13,11 @@ branches:
     - master
 
 install:
-  - # TODO: this is temporary, remove it when django-freeradius 0.1 is released
+  # TODO: this is temporary, remove it when django-freeradius 0.1 is released
   - pip install https://github.com/openwisp/django-freeradius/tarball/master
+  # Temporary: remove when openwisp-utils#71 is merged
+  - pip install "https://github.com/waleko/openwisp-utils/tarball/issues/40" --upgrade
```

Comment:
I released openwisp-utils 0.4.1 which contains your patch openwisp/openwisp-utils#71.
You can now remove this temporary addition.